src/System.Diagnostics.TextWriterTraceListener/tests/XmlWriterTraceListenerTests.cs

```diff
@@ -0,0 +1,501 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.CodeDom;
```

Comment:
any unneeded here?

Reply:
Ha, yes -- I was doing some local testing to write the tests 😆